### PR TITLE
Code Review of Q1

### PR DESCRIPTION
**Pull request by Harshwardhanwtagline**

### CODE REVIEW

File: q1.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+def numbers_operation(user_input, number_list):
+    if user_input == "A" or user_input == "a":
```
Comment (himanshu-suthar-tagline) [inline q1.py:2]: You can use here in condition like `user_input.lower() == 'a'` instead of writing both.

File: q1.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        return "Length of the List : {}".format(len(number_list))
+    elif user_input == "B" or user_input == "b":
+        return "First 3 Numbers: {}".format(number_list[0:3])
+    elif user_input == "C" or user_input == "c":
+        odd_sum = sum([i for i in number_list if i % 2 != 0])
+        return "Sum of odd Number: {}".format(odd_sum)
+    elif user_input == "D" or user_input == "d":
+        dict_1 = {i: number_list.count(i) for i in number_list}
```
Comment (himanshu-suthar-tagline) [inline q1.py:10]: Please use a good name for the variable instead of `i`

File: q1.py (hunk 1 above)
Comment (Harshwardhanwtagline) [reply q1.py:2]: Okay Sir i Have Change the Condition

File: q1.py (hunk 2 above)
Comment (Harshwardhanwtagline) [reply q1.py:10]: Yes i have replace the variable name

**Pull request merged**